index.php

```diff
@@ -24,5 +24,7 @@
 *  International Registered Trademark & Property of PrestaShop SA
 */
 
+// Meow !
```

Comment:
Regarding the LOLCODE (http://en.wikipedia.org/wiki/LOLCODE#Example_2), you should write ```BTW "Meow !"```